docs/source/data.rst

```diff
@@ -108,7 +108,44 @@ Data
 3D Cartographer Backpack – Deutsches Museum
 ===========================================
 
-3D data sets are still being curated and will be made available soon.
+This data was collected using a 3D LIDAR backpack at the
+`Deutsches Museum <https://en.wikipedia.org/wiki/Deutsches_Museum>`_.
+Each bag contains data from an IMU and data from two Velodyne VLP-16 LIDAR,
```

Comment:
Each bag contains data from an IMU and from two Velodyne VLP-16 LIDARs, one mounted horizontally (i.e. spin axis up) and one vertically (i.e. push broom).